### ISSUE TITLE
Prevent Removed Songs From Entering Que

### ISSUE THREAD

**Issue by RobGoldy** [dubtrack/www-dubtrack-fm#205]
Ayo,

I just woke up and went to que up a few songs and I thought of this feature. 

When Dubtrack adds songs from a playlist to the que (even if you que the entire playlist), it should check to see if the youtube/soundcloud video still exists. If the video doesn't exist, it should not que the video and skip over it. I feel as if this would prevent people from unknowingly quing songs that no longer work.

**Comment by Skavau** [comment]
Could cause huge lag as the system checks for it. Okay if you're queuing 3 songs but bad if its 200+ or something.

I understand the point though. It might not help matters though because as it doesn't queue the deleted songs, the user never becomes aware that they aren't queuing so they don't delete them or check for a new link.

**Comment by RobGoldy** [comment]
I understand that there needs to be a better way to address it such as making the song appear a different color and unable to be que so that the user will notice it.

I think that is a separate issue though and that the optimal solution would be that when a song is played but it doesn't exist, the system can recognize it (LIKE WHAT BOTS DO) and go to the next song in the user's que or skip it entirely. This way would not cause lag as it is only 1 time check per song played.

**Comment by nambok** [comment]
@RobGoldy yeah agreed, shouldnt really create much load. Ill add this to the roadmap

**Comment by Skavau** [comment]
A little notification for the user when a deleted song enters the room queue would work.

**Comment by Larry1123** [comment]
This would be a vary nice to have added.
